utils/appliance/implementations/ui.py

```diff
@@ -137,9 +137,11 @@ def create_view(self, *args, **kwargs):
 
     def am_i_here(self):
         try:
-            return self.view.is_displayed
-        except (AttributeError, NoSuchElementException):
+            self.view.assert_is_displayed()
```

Comment:
The problem is that there is a convention about that widgets (of which view are a subclass) provide `is_displayed` ... I was thinking, maybe we could create a decorator that would be supplied instead of property for the is_displayed that would unify this.